#define LEFT_ELBOW_CONTROLLER "left_elbow_position_controller"
#define LEFT_SHOULDER_CONTROLLER "left_shoulder_position_controller"
#define RIGHT_ELBOW_CONTROLLER "right_elbow_position_controller"
#define RIGHT_SHOULDER_CONTROLLER "right_shoulder_position_controller"

#define CMD_START_LEFT_ELBOW "START_LEFT_ELBOW_CTRL"
#define CMD_START_LEFT_SHOULDER "START_LEFT_SHOULDER_CTRL"
#define CMD_START_RIGHT_ELBOW "START_RIGHT_ELBOW_CTRL"
#define CMD_START_RIGHT_SHOULDER "START_RIGHT_SHOULDER_CTRL"

#define RESPONSE_LEFT_ELBOW "LEFT_ELBOW_CTRL_OK"
#define RESPONSE_LEFT_SHOULDER "LEFT_SHOULDER_CTRL_OK"
#define RESPONSE_RIGHT_ELBOW "RIGHT_ELBOW_CTRL_OK"
#define RESPONSE_RIGHT_SHOULDER "RIGHT_SHOULDER_CTRL_OK"

// 2020 - Pelekoudas I.
#define LEFT_ELBOW_EFFORT_CONTROLLER "left_elbow_effort_controller"
#define LEFT_SHOULDER_EFFORT_CONTROLLER "left_shoulder_effort_controller"
#define RIGHT_ELBOW_EFFORT_CONTROLLER "right_elbow_effort_controller"
#define RIGHT_SHOULDER_EFFORT_CONTROLLER "right_shoulder_effort_controller"

#define CMD_SWITCH_TO_EFFORT "SWITCH_ARM_TO_EFFORT_CONTROLLERS"
